src/ControlzEx/ControlzEx.csproj

```diff
@@ -16,9 +30,12 @@
     </PropertyGroup>
     <!-- Items include -->
     <ItemGroup>
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All"/>
```

Comment:
Do we need this if we use EmbedAllSources?

Reply:
That I cannot answer, that was suggested by @punker76

Reply:
If we now use `<EmbedAllSources>true</EmbedAllSources>` then we can remove the `SourceLink` reference.

Reply:
I always thought there is a huge difference between placing all the sources in the dll, vs. linking the symbols to a github repository... Is that really what you want?

Reply:
I'm not sure. `EmbedAllSources` is new to me and until yet I used gitlink or SourceLink. EmbedAllSources has the benefit of faster debugging, but this is not importend for me, I can live with gitlink or SourceLink.

Reply:
Your decision, but I would like to be able to continue my dotnet core 3.0 work with other repositories... so I hope we can close this PR soon :question: